src/test/java/org/jabref/logic/externalfiles/LinkedFileHandlerTest.java

```diff
@@ -0,0 +1,74 @@
+package org.jabref.logic.externalfiles;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.LinkedFile;
+import org.jabref.model.metadata.MetaData;
+import org.jabref.preferences.FilePreferences;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class LinkedFileHandlerTest {
+    private LinkedFileHandler linkedFileHandler;
+    private FilePreferences filePreferences;
+
+    @BeforeEach
+    void setUp(@TempDir Path testFolder) throws IOException {
+        Path bibPath = testFolder.resolve("bib_test.bib");
+        Path pdfPath = testFolder.resolve("pdf_test.pdf");
+        Files.createFile(bibPath);
+        Files.createFile(pdfPath);
+        try (OutputStreamWriter outputStreamWriter = new OutputStreamWriter(new FileOutputStream(bibPath.toAbsolutePath().toString()), StandardCharsets.UTF_8)) {
```

Comment:
Please use `Files.writeString` (see https://howtodoinjava.com/java11/write-string-to-file/)

Reply:
OK, I will have try.